package.json

```diff
@@ -1,132 +1,100 @@
 {
   "name": "@june-so/analytics-next",
-  "version": "1.36.5",
+  "version": "2.0.0",
```

Comment:
This is gonna be a major version bump because the code changes from

```javascript
let [response] = await AnalyticsBrowser.load({
  writeKey: 'abjq0PnNmeh7oQl9',
});
```

to

```javascript
let analytics = AnalyticsBrowser.load({
  writeKey: 'abjq0PnNmeh7oQl9',
});
```